crates/ursa-network/src/tests/service_tests.rs

```diff
@@ -61,21 +61,20 @@ mod tests {
         }
     }
 
-    async fn _run_bootstrap(
+    async fn run_bootstrap(
```

Comment:
we don't need to start two nodes for this test; one is fine to test that the network `inits`